maui-toc.html

```diff
@@ -157,6 +157,7 @@
                     <li><a href="/maui/cartesian-charts/line">Line Chart</a></li>
                     <li><a href="/maui/cartesian-charts/fastline">FastLine Chart</a></li>
                     <li><a href="/maui/cartesian-charts/area">Area Chart</a></li>
+			  <li><a href="/maui/cartesian-charts/boxAndWhisker">BoxAndWhisker Chart</a></li>
```

Comment:
change as Box and Whisker Chart
 "<a href="/maui/cartesian-charts/boxAndWhisker">Box and Whisker Chart</a>"